### DISCUSSION TITLE
Clone project, when compiling in github, server, api and other information fill in variables, compiled client does not take effect

### DISCUSSION THREAD

**Discussion by Timps123** [rustdesk/rustdesk#10194] [General]
Clone project, when compiling in github, server, api and other information fill in the variable, the compiled client does not take effect, is the variable name changed?

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/763, go to [discord](https://discord.gg/nDceKgxnkV) please, search there first, some users might have asked similar questions. Usually nobody will help you with build or self-hosting here.
